models/ClimateSensor.json

```diff
@@ -0,0 +1,86 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azsamples:iotdd:ClimateSensor;1",
```

Comment:
For our @ids, we should probably establish a basic format. The DTMI spec ([https://github.com/Azure/digital-twin-model-identifier](https://github.com/Azure/digital-twin-model-identifier)) suggests the reverse order of domain segments, so for Azure, dtmi:com:azure:...

Reply:
These IDs are subject to change. I started with `dtmi:com:azure:iotsamples` but then I found that the model repo does not allow to submit models with reserved words in the ID.

I'm ok with any suggestion.

Reply:
I guess we could use dtmi:com:example:... since the example.com domain name is reserved for examples.

---
In reply to: [417691842](https://github.com/rido-min/pnp-ppr-sample-spec/pull/2#discussion_r417691842) [](ancestors = 417691842)